controllers/registry.go

```diff
@@ -37,6 +37,7 @@ const (
 	RegistryStorageS3AccesskeyEnvVarKey      = "REGISTRY_STORAGE_S3_ACCESSKEY"
 	RegistryStorageS3BucketEnvVarKey         = "REGISTRY_STORAGE_S3_BUCKET"
 	RegistryStorageS3RegionEnvVarKey         = "REGISTRY_STORAGE_S3_REGION"
+	RegistryStorageS3ProfileEnvVarKey        = "REGISTRY_STORAGE_S3_PROFILE"
```

Comment:
Added this for consistency. Not really required.

Reply:
@deepakraj1997 As this env var is not actually required or consumed by the registry deployment can we get rid of it ? IMO It's presence might give a false sense that it is being used somewhere for configuration purposes.

Reply:
Thanks for reviewing. Removed.